lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3329,6 +3331,15 @@ def test_get_historical_coupon_codes(self):
             )
             coupon.save()
 
+        #now create coupons with the expiration dates
+        for i in range(5):
+            coupon = Coupon(
+                code='coupon{0}'.format(i), description='test_description', course_id=self.course.id,
+                percentage_discount='{0}'.format(i), created_by=self.instructor, is_active=True,
+                expiration_date=datetime.datetime.now(pytz.UTC) + datetime.timedelta(days=2)
+            )
+            coupon.save()
+
         response = self.client.get(get_coupon_code_url)
         self.assertEqual(response.status_code, 200, response.content)
         self.assertEqual(response['Content-Type'], 'text/csv')
```

Comment:
Could we assert the contents of the response, to verify the new coupons are there, with expiration dates?